packages/cli/internal/signatures.js

```diff
@@ -0,0 +1,25 @@
+function getSignatureWithParameterNamesAndValues(contractName, functionName, functionParameters) {
```

Comment:
Using this utility to present a function with its signature (and optionally values) instead of just the function name.

Reply:
what a name! :)

Reply:
Lol, well, it is what the function does ^^